src/vs/workbench/browser/parts/activitybar/media/activityaction.css

```diff
@@ -179,8 +179,8 @@
 .monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-label.profile-activity-item {
 	height: 20px;
 	width: 32px;
-	margin: 14px 8px;
-	padding: 0px;
+	margin: 14px 8px !important;
```

Comment:
`.monaco-workbench .activitybar.right > .content :not(.monaco-menu) > .monaco-action-bar .action-label:not(.codicon)` selector below overridden these two rules because that one has higher priority.

That selector only applies to icons of extension contributed view containers, but I tested icons in both SVG and PNG format, and it doesn't seem to do anything. However, I still didn't remove it but overridden them again with `!important`

Reply:
I think we should review these changes again because I'm not in favour of `@important`